Riot/Modules/Analytics/Analytics.swift

```diff
@@ -213,6 +213,38 @@ import AnalyticsEvents
     }
 }
 
+@objc
+protocol E2EAnalytics {
```

Comment:
Just extracted that protocol to make testing of decryption failure tracker easier